src/App.tsx

```diff
@@ -1,10 +1,33 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 
-export const App: React.FC = () => (
-  <div className="App">
-    {/* Keep this input for tests */}
-    <input type="text" />
+export const App: React.FC = () => {
+  const [keyPressed, setKeyPressed] = useState('');
 
-    <p className="App__message">The last pressed key is [Enter]</p>
-  </div>
-);
+  const detectKeyDown = (e: KeyboardEvent) => {
```

Comment:
Seeing the word `detect` I expect it to add event listener or sth. Will do better this way, I believe
```suggestion
  const handleKeyDown = (e: KeyboardEvent) => {
```

Reply:
I'm better at naming children (I hope :D)